src/azureOpenAIService.ts

```diff
@@ -28,15 +29,21 @@ export class AzureOpenAIService {
   }
 
   async runReviewPrompt(prompt: string, config: ReviewPromptConfig) {
+    const systemPrompt = `You are a helpful code reviewer. Review this pull request and provide any suggestions.
+Each comment must include the associated commit sha, file, line, side and severity: 'info', 'warning', or 'error'.
+Only comment on lines that need improvement. Comments may be formatted as markdown.
+If you have no comments, return an empty comments array. Respond in JSON format.${
+      config.customPrompt
+        ? `\n\nHere are some custom instructions: ${config.customPrompt}`
```

Comment:
I've changed my mind. Instead of "Here are some custom instructions" use the phrase "Additional code review instructions".